src/commands/createFunction/FunctionSubWizard.ts

```diff
@@ -86,7 +90,25 @@ export class FunctionSubWizard {
                     break;
             }
 
-            if ((!template.isHttpTrigger && !template.isSqlBindingTemplate) && !canValidateAzureWebJobStorageOnDebug(context.language) && !await getAzureWebJobsStorage(context, context.projectPath)) {
+            switch (context.newDurableStorageType) {
+                case DurableBackend.Netherite:
+                    promptSteps.push(new EventHubsConnectionPromptStep(), new NetheriteEventHubNameStep());
+                    executeSteps.push(new EventHubsConnectionExecuteStep(), new NetheriteConfigureHostStep());
+                    break;
+                case DurableBackend.SQL:
+                    // Todo: Uncomment out in future PR
+                    // promptSteps.push(new SqlDatabaseConnectionPromptStep(), new SqlDatabaseListStep());
+                    // executeSteps.push(new SqlDatabaseConnectionExecuteStep());
+                    break;
+                case DurableBackend.Storage:
+                default:
+            }
+
+            if (
+                !await getLocalConnectionString(context, ConnectionKey.Storage, context.projectPath) &&   // If we already have a storage setup, don't re-prompt
```

Comment:
Would appreciate a second pair of eyes to make sure I preserved the logic properly here!  I did my best to make it more readable and to add comments where appropriate